src/Groups/types.jl

```diff
@@ -285,6 +291,13 @@ f1*f3
 """
 const SubPcGroupElem = BasicGAPGroupElem{SubPcGroup}
 
+function Base.hash(x::Union{PcGroupElem,SubPcGroupElem}, h::UInt)
+  G = full_group(parent(x))[1]
+  h = is_finite_order(x) ? hash(order(x), h) : h
+  h = hash(is_finite(G), hash(G, h))
```

Comment:
```suggestion
  h = hash(G, h)
```
IMO the `is_finite(G)` part should already be used in hashing `G`, so it would be redundant here. Changing this makes it easier to compose different hash methods